### PR TITLE
Small changes to PDF processing and code cleaning

### PR DESCRIPTION
**Pull request by SoftDrink**
- Fix keyword search by making it not case sensible
- Add error handler to Nice spider (same one as Who spider)
- Fixes some part of the Dockerfile
- gitignore the scrapyd related folders, as we now deploy a scrapyd instance
- Changes logging level on ignored requests to debug, as we don't need to se them

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,11 +19,18 @@ RUN apt-get install -qqqy \
...
+RUN locale-gen en_GB.UTF-8
+
+ENV LC_ALL=en_GB.UTF-8
+ENV LANG=en_GB.UTF-8
+ENV LANGUAGE=en_GB.UTF-8
 
 # Update pip
 RUN pip3 install -qqq --upgrade pip
-
+RUN rm -f /usr/bin/python && ln /usr/bin/python3 /usr/bin/python
```
Comment (annikaC) [inline Dockerfile]: could use venv here instead of installing packages globally? (just in case you wanna run other projects on the same box at some point) https://docs.python.org/3/library/venv.html#module-venv

File: pdf_parser/objects/PdfObjects.py [hunk 2]
```diff
@@ -185,16 +185,16 @@ def get_lines_by_keywords(self, keywords, context=0):
             lines.extend(page.lines)
         ac_automaton.make_automaton()
         for num, line in enumerate(lines):
-            for index, value in ac_automaton.iter(line.text):
+            for index, value in ac_automaton.iter(line.text.lower()):
```
Comment (nsorros) [inline pdf_parser/objects/PdfObjects.py]: was that the sneaky line that was producing the error?

File: wsf_scraping/spiders/nice_spider.py [hunk 3]
```diff
@@ -167,7 +195,7 @@ def save_pdf(self, response):
         is_pdf = response.headers.get('content-type', '') == b'application/pdf'
 
         if is_scraped(response.request.url):
-            self.logger.info(
+            self.logger.debug(
```
Comment (nsorros) [inline wsf_scraping/spiders/nice_spider.py]: why did you change that?

File: pdf_parser/pdf_parse.py [hunk 4]
```diff
@@ -110,7 +110,7 @@ def parse_pdf_document_pdftxt(document):
     """
 
     parsed_path = document.name.replace('.pdf', '.html')
-    os.system('pdftotext -q -bbox %s %s 1>/dev/null 2>/dev/null' % (
+    os.system('pdftotext -q -bbox \'%s\' \'%s\' 1>/dev/null 2>/dev/null' % (
```
Comment (nsorros) [inline pdf_parser/pdf_parse.py]: does `.format` not work? isn't this the recommended way to string formating?

File: pdf_parser/objects/PdfObjects.py (hunk 2 above)
Comment (SoftDrink) [reply pdf_parser/objects/PdfObjects.py]: Yep, here and `re.IGNORECASE` just under it too

File: pdf_parser/pdf_parse.py (hunk 4 above)
Comment (SoftDrink) [reply pdf_parser/pdf_parse.py]: It is. Will change that 👍

File: wsf_scraping/spiders/nice_spider.py (hunk 3 above)
Comment (SoftDrink) [reply wsf_scraping/spiders/nice_spider.py]: It created a lot of noise in the logs, for the default log level is INFO. Also, it seems more of a DEBUG level information, as it only logs when we already did download the pdf.
